.github/workflows/ci.yml

```diff
@@ -37,3 +37,6 @@ jobs:
 
       - name: Typecheck
         run: pnpm typecheck
+
+      - name: Test
+        run: pnpm test
```

Comment:
Adding (newly introduced) unit tests to CI

Reply:
<img width="958" alt="CI unit tests" src="https://github.com/user-attachments/assets/5c8b4e04-0374-423d-8906-83b42682f914" />